tests/wpt/web-platform-tests/html/semantics/forms/form-submission-0/resources/url-encoded.py

```diff
@@ -0,0 +1,5 @@
+def main(request, response):
+    if request.body == "foo=bara":
+        return [("Content-Type", "text/plain")], "OK"
+    else:
+        return [("Content-Type", "text/plain")], "FAIL"
```

Comment:
This doesn't work right now - it keeps on returning "Unknown content type (application/json)" to the iframe. Not sure what's wrong here...